components/issue-selector/issue-selector.jsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import NavLink from '../nav-link/nav-link.jsx';
+import Utility from '../../js/utility';
+
+const ISSUES = [
```

Comment:
let's tackle it in another ticket: https://github.com/mozilla/network-pulse/issues/596